client/src/routes/widgetModule/WidgetModule.tsx

```diff
@@ -95,6 +125,188 @@ export const WidgetModule: React.FC<Props> = observer(({ domainName }) => {
+
+      if (numAlias >= maxAlias) {
+        setProcessStatus({
+          type: ProcessStatusTypes.PROGRESS,
+          render: (
+            <BaseText>
+              {walletStore.isMetamaskAvailable
+                ? 'Waiting for a transaction to be signed'
+                : 'Sign transaction on mobile device'}
+            </BaseText>
+          ),
+        })
+
+        const removingAlias = publicAliases[0]
+
+        const newPublicAliases = publicAliases.filter(
+          (item) => item !== removingAlias
+        )
+
+        const setResult = await rootStore.easClient.setPublicAliases({
+          domainName,
+          aliases: newPublicAliases,
+          onTransactionHash: () => {
+            setProcessStatus({
+              type: ProcessStatusTypes.PROGRESS,
+              render: <BaseText>Waiting for transaction confirmation</BaseText>,
+            })
+          },
+        })
+
+        if (setResult.error) {
+          throw setResult.error
+        }
+
+        const delResult = await rootStore.easClient.deactivate({
+          domainName,
+          alias: removingAlias,
+          onTransactionHash: () => {
+            setProcessStatus({
+              type: ProcessStatusTypes.PROGRESS,
+              render: <BaseText>Waiting for transaction confirmation</BaseText>,
+            })
+          },
+        })
+
+        if (delResult.error) {
+          throw delResult.error
+        }
+
+        const { success, error } = await easServerClient.deactivate(
+          domainName,
+          removingAlias
+        )
+
+        if (error) {
+          throw new Error(error)
+        }
+      }
+
+      const signature = await rootStore.easClient.buildSignature(
+        domainName,
+        alias,
+        forward
+      )
+      const separator = ethers.utils.toUtf8Bytes(
+        await rootStore.easClient.getSeparator()
+      )
+      const data = ethers.utils.concat([
+        ethers.utils.toUtf8Bytes(alias),
+        separator,
+        ethers.utils.toUtf8Bytes(forward),
+        separator,
+        signature,
+      ])
+      let makePublic = true
+      const commitment = ethers.utils.keccak256(data)
+
+      console.log('### publicAliases', publicAliases)
+
+      if (publicAliases.includes(alias)) {
+        makePublic = false
+      }
+
+      setProcessStatus({
+        type: ProcessStatusTypes.PROGRESS,
+        render: (
+          <BaseText>
+            {walletStore.isMetamaskAvailable
+              ? 'Waiting for a transaction to be signed'
+              : 'Sign transaction on mobile device'}
+          </BaseText>
+        ),
+      })
+
+      const activateResult = await rootStore.easClient.activate({
+        domainName: domainName,
+        onTransactionHash: () => {
+          setProcessStatus({
+            type: ProcessStatusTypes.PROGRESS,
+            render: <BaseText>Waiting for transaction confirmation</BaseText>,
+          })
+        },
+        alias,
```

Comment:
this parameter should be bytes32

Reply:
https://github.com/harmony-one/1-country.frontend/pull/149#discussion_r1181971753

Reply:
okay